view/theme/frio/templates/jot-header.tpl

```diff
@@ -296,27 +296,29 @@
 	}
 
 	function itemFiler(id) {
-		const bordercolor = $("input").css("border-color");
+		var modal = $('#modal').modal();
 
 		$.get('filer/', function (data) {
-			$.colorbox({html: data});
-			$("#id_term").keypress(function () {
-				$(this).css("border-color", bordercolor);
-			})
-			$("#select_term").change(function () {
-				$("#id_term").css("border-color", bordercolor);
-			})
-
+			modal
+				.find('#modal-body')
+				.append(data);
+			
+			// Ensure focus after the modal is fully visible
+       		modal.on('shown.bs.modal', function () {
+            	$('#id_term').trigger('focus');
+        	});
+			
```

Comment:
The indentation isn't consistent in this file, please normalize to tabulations.

Reply:
Sorry was synced fo fast with upstream :-/ Still on it :-)